### PR TITLE
correct eslint error

### PR DESCRIPTION
**Pull request by DedrickEnc**
This P.R fixes some linting errors and warnings
Thank you for contributing!

Before submitting this pull request, please verify that you have:
 - [x] Run your code through eslint.
 - [x] Run integration tests.
 - [x] Run end-to-end tests.
 - [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the online review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

File: server/controllers/finance/reports/cash/index.js [hunk 1]
```diff
@@ -136,7 +136,12 @@ function receipt(req, res, next) {
 function report(req, res, next) {
   let reportInstance;
   const query = _.clone(req.query);
-  _.extend(query, { filename : 'TREE.CASH_PAYMENT_REGISTRY', csvKey : 'rows', footerRight : '[page] / [toPage]', footerFontSize : '7'});
+  _.extend(query, {
```
Comment (sfount) [inline server/controllers/finance/reports/cash/index.js:139]: :+1: This is much more readable!

**Review by sfount — APPROVED** [review]
Changes look good to me - small updates like this make the code much more readable and easier to understand. Good work!

**Comment by jniles** [comment]
This looks good to me.  I think that this will likely have a lot of conflicts if we merge it currently.  I'll schedule this for a merge after some of the other PRs have landed.

**Review by jniles — APPROVED** [review]

**Pull request merged**